.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: pass-docker-services Continuous Integration
+on: [ pull_request, workflow_dispatch ]
+
+concurrency:
+  group: ci-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  print-workflow-description:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "This is a CI build of branch ${{ github.ref }} in repository ${{ github.repository }}"
+      - run: echo "This job was triggered by a ${{ github.event_name }} event and is running on a ${{ runner.os }} server"
+
+  call-workaround:
+    name: Run Parent POM Workaround
+    uses: eclipse-pass/pass-indexer-checker/.github/workflows/parent-pom-workaround.yml@main
+  
+  call-tests-unit:
+    name: Run Unit Tests
+    uses: eclipse-pass/pass-indexer-checker/.github/workflows/tests-unit.yml@main
+    needs: call-workaround
+    
+#  call-tests-integration:
```

Comment:
Did we want to add a comment outlining the issue that is blocking the integration tests so that we know when to re-try (how to start to look at fixing)?

Reply:
Forcing the use of maven 3.8.4 (instead of the default 3.8.5) solves the integration tests failing. I have added this "fix" and put the integration tests back into play.